packages/cli/src/build/build.ts

```diff
@@ -216,6 +211,22 @@ const buildAndOutputNonComponentFiles = async (targetContext: TargetContextWithC
   return await outputNonComponentFiles({ ...targetContext, files });
 };
 
+function runHook(hook: string, config?: MitosisConfig) {
+  const noop = () => {};
+  const hookFn = get(config, `hooks.${hook}`);
```

Comment:
I would type this better, and remove the `lodash` dependency (I find them too bulky and the string can have any number of typos). Something like:

```suggestion
function runHook(hook: keyof MitosisConfig['hooks'], config?: MitosisConfig) {
  const noop = () => {};
  const hookFn = config?.hooks?.[hook]
```

Reply:
I have modified the plugin scheme